### DISCUSSION TITLE
6600 Console Glyphs

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#83] [General]
Where is the document defining the console font on the 6600?

Have any modern facsimiles of it been done?  TTF? Hershey format? Anything?

**Comment by philclaridge** [comment]
Page 26 on for 6602 console char gen.
http://bitsavers.informatik.uni-stuttgart.de/pdf/cdc/cyber/cyber_70/fieldEngr/60125000C_6602_6603_6622_6681_6682_Data_Channel_Diagrams_Dec65.pdf

Also see https://retrocomputing.stackexchange.com/questions/17115/cdc-6600-system-console for distortion added by X-Y analog deflection circuits.

I don’t know of a facsimile font.

I am (slowly) working on an updated console as well.

Phil

**Reply by KC5VDJ** [reply]
Hmm..  is that really the font as displayed on "Design of a Computer"?  The 'E' matches among others, assuming a slight defocus, but the '6's on the book cover have a bit more of a angle where the inside connects, but....given the guaranteed positioning accuracy being course...

Could the defocus have been to prevent burn-in, while providing a cathode / phosphor capable of 24/7/365 for a planned decade or two?  Another possible explanation could be that a thinner stroke might not always connect where it should visibly, the defocus thickening the line enough to make the point moot?

Maybe TTF with a thick stroke width, and slight shape imperfections...to do it right, I'd need like a fully as-displayed screen image containing all printable chars.  There has to be one somewhere, it would have had a similar camera as radar scopes.

**Comment by philclaridge** [comment]
Let me know if you solve the puzzle.  

I am looking around for 6600 screen shots. Here is the best I know of, about 3 min in: https://www.youtube.com/watch?v=wcSmUEaCRp8

**Reply by KC5VDJ** [reply]
Cool video.  Looks about the same timeframe as the NAG TRANSIT System video.  Used to work across the street from NAG (in Range Instrumentation Systems) as well.  Heard that NAG and Laguna Peak (big dish) recently transferred to Space Command.

**Comment by pa0gri** [comment]
Look at http://pa0gri.ampr.org/lcm_reunion/index.html photos 19-24 <http://pa0gri.ampr.org/lcm_reunion/index.html%20photos%2019-24>

**Reply by KC5VDJ** [reply]
Excellent!!!  Now to give my wife a task, she wants to play with a font creator!  This should do perfect, combined with the oscilloscope photograph used for the book ... The small is unreadable, but size can be derived, subtracting the intense halo, and slight blur, combined with the specs...

Someone needs to explain proper crt adjustment to the op though, either the stop on the camera was too wide, or the beam was way too strong on the smaller ones, I don't think it was the camera, halo is very evident.  Old CRTs like those need to be slightly under-driven...   Not much better than Willemite (Zn2SiO4:Mg 0.8 or something like that, Zinc Orthosilcate, Maganese Activated, First discovered in nature in Willem-something, PA, see RCA, Vacuum Tube Design, Luminescent Materials, Lecture 5, (1940)), thermal heating (amperage, see Sir Issac Newton) kills the phosphor.  How easy would it be to obtain a replacement crt?

RCA intentionally over-specified the target voltage on electron-ray indicators by the time of publication of "Vacuum Tube Design", already having three years of field experience (6T5, etc) at the time of the lectures, around the time tubes were getting reliable, thus sales down..  over-driven targets were bright and pretty, and made watching a radio less boring, and even helped sell enough radios to do their part to scratch out of the final days of the depression, but they'd have to buy two or three tubes a year with the 6U5, 6G5, 6E5, etc....  please don't over-drive those CRTs!

Sorry for the rant, but it's a pet peeve.  You should see the prices on the remaining new electron-ray indicators in the octal base, all because too many people restoring old radios didn't take the fact that the datasheet was greed-based into account...they were intended to be a short-lived consumer item, thus they didn't even label the datasheet "ICAS" ("Intermittent Amateur / Commercial Service", the tube equivalent of upping Vcore and the clock multiplier), the radio repairman wasn't going to say shit, how much did he make for each house call to replace a tube, his bread and butter...

**Reply by KC5VDJ** [reply]
Followup...  Just re-read that...  Now I'm wondering what color the waterfall was in "A-Plumbing We Will Go" (1940, Columbia Pictures)???  YELLOW?!?!?!  Dude here talks like he was working for Zvorykin ("Electron Optics" Parts 1-2 of 3, ibid.)
